tutorbackup/plugin.py

```diff
@@ -21,6 +21,8 @@
         "K8S_CRONJOB_BACKUP_SCHEDULE": "0 0 * * *",
         "K8S_CRONJOB_RESTORE_ENABLE": False,
         "K8S_CRONJOB_RESTORE_SCHEDULE": "30 0 * * *",
+        "K8S_USE_EPHEMERAL_VOLUMES": False,
+        "K8S_EPHEMERAL_VOLUME_SIZE": "1Gi",
```

Comment:
`tutor k8s` allocates `5Gi` to the MongoDB and MySQL PVs. Wouldn't it thus make sense to use `10Gi` as the default here?